doc/lua_api.txt

```diff
@@ -3911,12 +3911,16 @@ It can be created via `Settings(filename)`.
 #### Methods
 * `get(key)`: returns a value
 * `get_bool(key)`: returns a boolean
+* `get_np_group(key)`: returns a NoiseParams table
```

Comment:
\* Unlike `minetest.get_noiseparams`, the values in the table are converted to strings

Reply:
Nope, see https://github.com/minetest/minetest/commit/a1e1a19ac355dbe7e1a2e060fd6b5fe082298eb6#commitcomment-25125537